### DISCUSSION TITLE
mice imputation

### DISCUSSION THREAD

**Discussion by mehrdad123qaderi** [bioinformatics-ptp/detectRUNS#46] [General]
in this code:
SL.lib <- c("SL.mean")
imp.SL <- mice::mice( df_missing10, m = 5, maxit = 5,
method = "SuperLearner",
print = FALSE, SL.library = SL.lib)
i have error:
Error in names(newdata) <- sapply(1:ncol(newdata), function(n) { :
'names' attribute [2] must be the same length as the vector [0]

**Comment by bunop** [comment]
Same question posted on #45 . Seems to be a problem not related to `detectRUNS`, should be posted on [mice bugreports](https://github.com/amices/mice/issues) page
